config.json

```diff
@@ -41,7 +64,11 @@
             "when command is mist:ec2-tags must have mist:manage-tags",
             "when command is mist:s3-buckets must have mist:view",
             "when command is mist:s3-buckets with arg[0] == 'list' must have mist:view",
-            "when command is mist:s3-buckets with (arg[0] == 'delete' or arg[0] == 'rm') must have mist:destroy"],
+            "when command is mist:s3-buckets with (arg[0] == 'delete' or arg[0] == 'rm') must have mist:destroy",
```

Comment:
It's not a change; it's in addition too. `s3-buckets` only deals with buckets. `s3-bucket-list` deals with what's in the buckets. Maybe that's a bad name for the command. How about `s3-bucket_files`?